Tests/Editor/SettingsAnalysisTests.cs

```diff
@@ -116,6 +116,47 @@ public void SettingsAnalysis_SplashScreen_IsEnabledAndCanBeDisabled(bool splashS
             PlayerSettings.SplashScreen.show = prevSplashScreenEnabled;
         }
 
+        [Test]
+        public void SettingsAnalysis_AudioMode_SpeakerModeStereo_IsReported()
+        {
+            var audioConfiguration = AudioSettings.GetConfiguration();
+            AudioSettings.speakerMode = AudioSpeakerMode.Stereo;
+
+            var issues = Analyze(IssueCategory.ProjectSetting, i => i.descriptor.id.Equals("PAS0033"));
+            var playerSettingIssue = issues.FirstOrDefault();
+
+            Assert.NotNull(playerSettingIssue);
+            
+            AudioSettings.Reset(audioConfiguration);
+        }
+
+        [TestCase(false)]
+        [TestCase(true)]
+        public void SettingsAnalysis_AudioMode_IsSpeakerModeMono(bool isSpeakerMonoMode)
+        {
+            var audioSpeaker = AudioSettings.GetConfiguration();
+            AudioSettings.speakerMode =
```

Comment:
does this result in compile warnings to the unsupported AudioSettings.speakerMode?